mimic/rest/maas_api.py

```diff
@@ -118,7 +118,7 @@ def create_entity(params):
     """
     params = collections.defaultdict(lambda: '', params)
     newentity = {}
-    newentity['label'] = params[u'label'].encode("utf-8")
+    newentity['label'] = params['label']
```

Comment:
This points at a weakness in the `create_entity` docstring: what is it returning a dictionary of? What are the types of the keys and values?  (I think it's clear that the right answer is "unicode" but it should really say so.)